requirements.txt

```diff
@@ -2,5 +2,6 @@
 requests<2.15,>=2.2
 pyxdg
 dnspython
+future
```

Comment:
Actually, where is this being used?  You're not importing `future`, and you're not importing `builltins`, and you've try/except blocks for Python 2 vs Python 3 names in multiple places... What am I missing?

Reply:
`from __future__ import print_function`
But anyway, it will be required on many places in following Python 3 compatibility commits.